packages/nextra/src/mdx-plugins/remark.ts

```diff
@@ -37,7 +37,8 @@ export const remarkHeadings: Plugin<[], Root> = function (this: Processor) {
               (child: { type: string }) => child.type === 'mdxJsxTextElement'
             )
           const heading = {
-            ...(node as Heading),
+            type: node.type,
```

Comment:
Do we need `type`? I guess it always `type: heading`?

Reply:
Yeah I think we can simplify this!